addons/clouds/clouds.gdshader

```diff
@@ -167,7 +167,7 @@ vec4 march(vec3 pos,  vec3 end, vec3 dir, int depth) {
 	vec3 p = pos + hash(pos * 10.0) * ss;
 	const float t_dist = sky_t_radius-sky_b_radius;
 	float lss = (t_dist / 36.0);
-	vec3 ldir = normalize(LIGHT0_DIRECTION);
+	vec3 ldir = normalize(LIGHT0_DIRECTION) * 0.6;
```

Comment:
Maybe add a description of why we did this so we recognize it in the future, e.g. `// Multiply by 0.6 to make the sun appear darker behind the clouds instead of creating a large purely white area in the sky (trick to simulate that the sun is always at a larger angle to the viewer than it actually is)`

Reply:
[a58a139](https://github.com/boku-ilen/landscapelab/pull/326/commits/a58a139a83e24aea7f5f4fea0abec57cef8c9992)